gamemodes/terrortown/entities/entities/ttt_traitor_button/init.lua

```diff
@@ -16,14 +16,15 @@ local path = dir .. "/" .. game.GetMap() .. ".json"
 local function ReadMapConfig()
 	file.CreateDir(dir)
 
-	local modTime = (not file.Exists(path, "DATA") and (lastRead + 1)) or file.Time(path, "DATA")
+	local modTime = (not file.Exists("data_static/" .. path, "GAME") and (lastRead + 1)) or file.Time("data_static/" .. path, "GAME")
+	modTime = modTime <= lastRead and (not file.Exists(path, "DATA") and (lastRead + 1)) or file.Time("data_static/" .. path, "DATA")
```

Comment:
There should be no `data_static/` here to retain the old behaviour, correct?

```suggestion
	modTime = modTime <= lastRead and (not file.Exists(path, "DATA") and (lastRead + 1)) or file.Time(path, "DATA")
```